doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -321,6 +330,10 @@ Section :ref:`explicit-applications`).
 Type cast
 ---------
 
+.. index::
+   single: ... : ... (type cast)
+   single: ... <: ... (type cast)
```

Comment:
Can you also add `... <<: ...`? Shouldn't we explain what those casts mean, though?

Reply:
Of course we should. See #7846.